apps/researcher/src/messages/nl/messages.json

```diff
@@ -91,8 +91,11 @@
     "providerCurrentHolder": "Data provider of huidig object bewaarder",
     "providerInfo": "Meer info",
     "contactInfo": "Contact gegevens",
-    "linkToWebsite": "Website",
-    "linkToDataset": "Meer over de dataset",
+    "dataProviderTitle": "Data provider of huidig object bewaarder",
```

Comment:
Or: `Bronhouder object` (the term that we use in the facets) or just `Bronhouder`?